ticketutil/servicenow.py

```diff
@@ -0,0 +1,323 @@
+        except requests.RequestException as e:
+            logging.error("Error editing ticket")
+            logging.error(e.args[0])
+
+    def add_comment(self, comment):
+        """
+        Adds comment
+
+        :param comment: new ticket comment
+        """
+        try:
+            logging.info('Adding comment')
+            self.edit(comments = comment)
+        except:
+            logging.error('Failed to add the comment')
+
+    def add_cc(self, user):
+        """
+        Adds user(s) to cc list.
+        :param user: A string representing one user's email address, or a list
+        of strings for multiple users.
+        :return:
+        """
+        try:
+            watch_list = self.ticket_content['watch_list'].split(',')
+            watch_list = [item.strip() for item in watch_list]
+            if isinstance(user, str):
+                user = [user]
+            for item in user:
+                if item not in watch_list:
+                    watch_list.append(item)
+
+            logging.info('Adding user(s) to CC list')
+            watch_list = ', '.join(watch_list)
+            self.edit(watch_list=watch_list)
+        except:
+            logging.error('Failed to add user(s) to CC list')
+
+    def rewrite_cc(self, user):
+        """
+        Rewrites user(s) in cc list.
+        :param user: A string representing one user's email address, or a list
+        of strings for multiple users.
+        :return:
+        """
+        try:
+            if isinstance(user, str):
+                user = [user]
+            logging.info('Rewriting CC list')
+            watch_list = ', '.join(user)
+            self.edit(watch_list=watch_list)
+        except:
+            logging.error('Failed to rewrite CC list')
+
+    def remove_cc(self, user):
+        """
+        Removes user(s) from cc list.
+        :param user: A string representing one user's email address, or a list of strings for multiple users.
+        :return:
+        """
+        try:
+            watch_list = self.ticket_content['watch_list'].split(',')
+            watch_list = [item.strip() for item in watch_list]
+            if isinstance(user, str):
+                user = [user]
+            for item in user:
+                print(item)
+                if item in watch_list:
+                    watch_list.remove(item)
+
+            logging.info('Removing user(s) from CC list')
+            watch_list = ', '.join(watch_list)
+            self.edit(watch_list=watch_list)
+        except:
+            logging.error('Failed to remove user(s) from CC list')
+
+    def _prepare_ticket_fields(self, fields):
+        """
+        Makes sure each key value pair in the fields dictionary is in the correct form.
+        :param fields: Ticket fields.
+        :return: fields: Ticket fields in the correct form for the ticketing tool.
+        """
+        if 'urgency' not in fields:
```

Comment:
That's what I intended, but now that I think about it might not have been such a good idea. Adding these two fields may rewrite existing urgency and impact. They are mandatory by PNT DevOps One application for ServiceNow, but the API should be able to create a ticket even without them. So I am removing these for the sake of less complex code.